README.md

```diff
@@ -91,72 +86,48 @@ As a root launch HAProxy with configuration:
 ```
 $ haproxy -f config.cfg
 ```
-Create Global Config, see example in [examples/cfg/] (examples/cfg/).
 
-In one terminal window, open the snap daemon (in this case with logging set to 1,  trust disabled and global configuration saved in cfg.json):
+Download an [example Snap global config](https://github.com/intelsdi-x/snap-plugin-collector-haproxy/blob/master/examples/cfg/snap-global-cfg.json) file.
 ```
-$ snapd -l 1 -t 0 --config examples/cfg/snap-global-cfg.json
+$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-haproxy/master/examples/cfg/snap-global-cfg.json
 ```
-In another terminal window in snap-plugin-collector-haproxy directory:
-Load haproxy plugin:
+
+Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap) with provided configuration file:
+* command line: `snapd -l 1 -t 0 --config cfg.json&`
+
+Download and load Snap plugins:
 ```
-$ snapctl plugin load build/rootfs/snap-plugin-collector-haproxy
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-haproxy/latest/linux/x86_64/snap-plugin-collector-haproxy
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/linux/x86_64/snap-plugin-publisher-file
+$ chmod 755 snap-plugin-*
+$ snapctl plugin load snap-plugin-collector-haproxy
+$ snapctl plugin load snap-plugin-publisher-file
 ```
+
 See available metrics for your system
 ```
 $ snapctl metric list
 ```
-Create a task manifest file (exemplary file in [examples/tasks/] (examples/tasks/)):
-```
-{
-  "version": 1,
-  "schedule": {
-    "type": "simple",
-    "interval": "1s"
-  },
-  "workflow": {
-    "collect": {
-      "metrics": {
-        "/intel/haproxy/*": {}
-      },
-      "config": null,
-      "publish": [
-        {
-          "plugin_name": "file",
-          "config": {
-            "file": "/tmp/published_haproxy.log"
-          }
-        }
-      ]
-    }
-  }
-}
-```
-
-Get mock file plugin for publishing, appropriate for Linux or Darwin:
-```
-wget  http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/linux/x86_64/snap-plugin-publisher-file
-```
-or 
-```
-wget  http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/darwin/x86_64/snap-plugin-publisher-file
-```
 
-Load mock file plugin for publishing:
+Download an [example task file](https://github.com/intelsdi-x/snap-plugin-collector-haproxy/blob/master/examples/tasks/task.json) and load it:
```

Comment:
To make the link work from any commit/fork, replace the url to `examples/tasks/task.json`.